### ISSUE TITLE
No module named 'werkzeug.contrib'

### ISSUE THREAD

**Issue by MariumAZ** [HDI-Project/ATMSeer#50]
When I tried ATMSeer using docker-compose I had :  ImportError: No module named 'werkzeug.contrib'

**Comment by micahjsmith** [comment]
Hi @MariumAZ can you please provide any additional information to help us reproduce the error?

```
## Steps to Reproduce the Problem

  1.
  1.
  1.

## Specifications

  - ATMSeer Version:
  - Vagrant Version:
  - VirtualBox Version:
  - OS Version:
```

**Comment by MariumAZ** [comment]
Hi @micahjsmith  I tried to open ATMSeer on chrome so I used  :  **sh install-docker.sh**
However then I had a problem . 
![atmseer](https://user-images.githubusercontent.com/49347660/84167149-244b5100-aa76-11ea-8508-1516a27e4985.PNG)

**Comment by MariumAZ** [comment]
@micahjsmith 
 Steps to Reproduce the Problem:

  1.git clone https://github.com/HDI-Project/ATMSeer.git
  2.sh install-docker.sh

**Comment by micahjsmith** [comment]
Thank you! We will look into this

**Comment by micahjsmith** [comment]
Fixed by cac1617

There are still issues installing with Docker, will track in another issue
